### ISSUE TITLE
NOTICE: undefined offset

### ISSUE THREAD

**Issue by zedzedzed** [zedzedzed/table-of-contents-plus#38]
**Reported by michael on 31 Oct 2011 10:17 UTC**
Pages hosted at http://two.dev/ seem to produce these notices, eg http://two.dev/gov-2-primer/

Lines 924, 925

**Comment by zedzedzed** [comment]
**Comment by michael on 31 Oct 2011 11:13 UTC**
Those lines already apply the @ symbol to suppress errors but according to http://php.net/manual/en/language.operators.errorcontrol.php, it should only be used on expressions. Both uses do not appear to be expressions.

**Comment by zedzedzed** [comment]
**Comment by michael on 31 Oct 2011 11:28 UTC**
In [120]:
```CommitTicketReference repository="" revision="120"
closes #40
```

**Comment by zedzedzed** [comment]
**Modified by michael on 31 Oct 2011 11:41 UTC**

**Comment by zedzedzed** [comment]
**Comment by michael on 2 Nov 2011 13:03 UTC**
Other notices (while saving admin options):

Notice: Undefined variable: post_id in  /data/www/two.dev/wp-content/plugins/table-of-contents-plus/toc.php on  line 340

Notice: Undefined index: custom_links_visited_colour in  /data/www/two.dev/wp-content/plugins/table-of-contents-plus/toc.php on  line 352

Notice: Undefined index: bullet_spacing in  /data/www/two.dev/wp-content/plugins/table-of-contents-plus/toc.php on  line 377

Notice: Undefined index: include_homepage in  /data/www/two.dev/wp-content/plugins/table-of-contents-plus/toc.php on  line 378

**Comment by zedzedzed** [comment]
**Comment by michael on 2 Nov 2011 13:16 UTC**
In [121]:
```CommitTicketReference repository="" revision="121"
re #40 addresses more NOTICES
closes #19 now supports repeated headings
```

**Comment by zedzedzed** [comment]
**Comment by michael on 6 Nov 2011 00:57 UTC**
In [127]:
```CommitTicketReference repository="" revision="127"
closes #41
re #40 more notices knocked off
```
